### PR TITLE
Freshen up subecosystems of Moonriver

### PR DESCRIPTION
**Pull request by themacexpert**
This PR freshens a variety of ecosystems, adding missed repos and removing stale ones. This PR does not modify what is considered a subecosystem of Moonriver network.

### CODE REVIEW

**Comment by themacexpert** [comment]
@eherrerosj Hey, is there anything I need to fix here?

**Review by eherrerosj — COMMENTED** [review]
nice PR, left a couple comments

File: data/ecosystems/b/bware-labs.toml [hunk 1]
```diff
@@ -9,12 +9,6 @@ github_organizations = ["https://github.com/bwarelabs"]
 [[repo]]
 url = "https://github.com/bwarelabs/AIPs"
 
-[[repo]]
-url = "https://github.com/bwarelabs/aptos-core"
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply data/ecosystems/b/bware-labs.toml:13]: Not sure, I added it back

File: data/ecosystems/o/onfinality.toml [hunk 2]
```diff
@@ -6,11 +6,84 @@ sub_ecosystems = []
 github_organizations = []
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply data/ecosystems/o/onfinality.toml]: Done

**Comment by eherrerosj** [comment]
hey @themacexpert, you think we could add a missing tag instead of removing repos that don't exist anymore publicly? We prefer to keep non-existant repos so we know they existed at some point and we can keep their commits. 

Example: https://github.com/electric-capital/crypto-ecosystems/blob/master/data/ecosystems/h/helios-labs.toml#L11

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left another comment

**Comment by themacexpert** [comment]
@eherrerosj Done! The only remaining deletions are instances where repos have been renamed.

**Comment by themacexpert** [comment]
Any other updates I need to make here? Thank you @eherrerosj

**Comment by eherrerosj** [comment]
> Any other updates I need to make here? Thank you @eherrerosj

LGTM now, thank you for the work @themacexpert !

**Pull request merged**